validation/exclusions_test.go

```diff
@@ -56,8 +56,37 @@ var exclusions = map[string]map[uint64]bool{
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
 	},
 	"Standard_Contract_Versions": {
-		11155421: true, // sepolia-dev-0/oplabs-devnet-0
-		11763072: true, // sepolia-dev-0/base-devnet-0
+		10:        true, // mainnet/op           MCP (at time of writing)
+		8453:      true, // mainnet/base         MCP (at time of writing)
+		957:       true, // mainnet/lyra         MCP (at time of writing)
+		1750:      true, // mainnet/metal        MCP (at time of writing)
+		34443:     true, // mainnet/mode         MCP (at time of writing)
+		291:       true, // mainnet/orderly      MCP (at time of writing)
+		424:       true, // mainnet/pgn          MCP (at time of writing)
+		7777777:   true, // mainnet/zora         MCP (at time of writing)
+		11155420:  true, // sepolia/op           MCP (at time of writing)
+		84532:     true, // sepolia/base         MCP (at time of writing)
+		919:       true, // sepolia/mode         MCP (at time of writing)
+		58008:     true, // sepolia/pgn          MCP (at time of writing)
+		999999999: true, // sepolia/zora         MCP (at time of writing)
+		11155421:  true, // sepolia-dev0/oplabs-devnet-0"
+		11763072:  true, // sepolia-dev0/base-devnet-0"
```

Comment:
Those 2 have FPs, right?

Reply:
This check isn't just for "having FPs" but for having specific versions of contracts deployed. `sepolia-dev0/oplabs-devnet-0` has: 
```
                    {
                                 "L1CrossDomainMessenger": "2.1.1",
                                 "L1ERC721Bridge": "1.5.0",
                                 "L1StandardBridge": "2.0.0",
                                 "L2OutputOracle": "",
                                 "OptimismMintableERC20Factory": "1.8.0",
                                 "OptimismPortal": "3.10.0",
                                 "SystemConfig": "1.11.0",
                                 "ProtocolVersions": "",
                                 "SuperchainConfig": "",
                                 "AnchorStateRegistry": "0.1.0",
                                 "DelayedWETH": "0.2.0",
                                 "DisputeGameFactory": "1.0.0",
                                 "FaultDisputeGame": "1.2.0",
                                 "MIPS": "0.1.0",
                                 "PermissionedDisputeGame": "1.2.0",
                                 "PreimageOracle": "1.0.0"
                                }
```

https://github.com/ethereum-optimism/superchain-registry/blob/8422aca2afeed0197fdc0fca104b80e707a4fa2b/validation/standard/standard-versions.toml#L1-L15